.github/pull_request_template.md

```diff
@@ -2,11 +2,12 @@ Feel free to open a PR before all of these items have been completed.
 
 Pull Request Checklist
  - [ ] Pull request includes a description of the change and the reason behind it.
- - [ ] Pull request [uses keywords to close relevant issues](https://help.github.com/en/articles/closing-issues-using-keywords).
+ - [ ] Pull request [uses keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to close relevant [issues](https://github.com/uptake/autofocus/issues).
  - [ ] Pull request includes unit tests for any new functionality.
  - [ ] README and docs have been updated.
- - [ ] `./.ci/test.sh` passes locally.
+ - [ ] `./.ci/local_checks.sh` passes locally. (The app must be running. See `README.md` for instructions.)
 
 Maintainer's responsibilities:
 - [ ] `_version.py` has been updated.
 - [ ] `CHANGELOG.md` has been updated.
+- [ ] Updated app container has been pushed, if relevant.
```

Comment:
@jameslamb would I do e.g. `docker build . -t gsganden/autofocus:1.2.1` to version the docker container?

Reply:
yep!